playbooks/common/openshift-checks/health.yml

```diff
@@ -1,14 +1,9 @@
 ---
-- include: ../openshift-cluster/std_include.yml
-  tags:
-  - always
-
 - name: Run OpenShift health checks
-  hosts: OSEv3
+  hosts: oo_all_hosts
   roles:
-  - openshift_health_checker
-  vars:
-  - r_openshift_health_checker_playbook_context: health
+  - role: openshift_health_checker
+    r_openshift_health_checker_playbook_context: health
```

Comment:
I tested this pattern using the `adhoc.yml` playbook from #4570 and found out that it breaks the intended behavior.

We use this variable to communicate in a rather unorthodox way with the callback plugin to print a summary of errors.
For the solution to work, the variable must be visible to the action plugin call in the `post_tasks` section.
When I set the variable only for the role include, the callback plugin comes with `context=None`.